.changeset/witty-bees-agree.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/admin.server-configurations.v1": patch
```

Comment:
let's patch bump console too so that a new console version with this change will be released

Reply:
Added